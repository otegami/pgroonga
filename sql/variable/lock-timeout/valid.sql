-- To load PGroonga
SELECT pgroonga_command('status')::json->0->0;

SHOW pgroonga.lock_timeout;
SET pgroonga.lock_timeout = 1000;
SHOW pgroonga.lock_timeout;
SET pgroonga.lock_timeout = default;
SHOW pgroonga.lock_timeout;
