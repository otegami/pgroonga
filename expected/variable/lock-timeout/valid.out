-- To load PGroonga
SELECT pgroonga_command('status')::json->0->0;
 ?column? 
----------
 0
(1 row)

SHOW pgroonga.lock_timeout;
 pgroonga.lock_timeout 
-----------------------
 900000
(1 row)

SET pgroonga.lock_timeout = 1000;
SHOW pgroonga.lock_timeout;
 pgroonga.lock_timeout 
-----------------------
 1000
(1 row)

SET pgroonga.lock_timeout = default;
SHOW pgroonga.lock_timeout;
 pgroonga.lock_timeout 
-----------------------
 900000
(1 row)

